src/components/Settings/Speech/Speech.container.js

```diff
@@ -73,14 +87,16 @@ export class SpeechContainer extends Component {
     this.updateSettingsTimeout = setTimeout(async () => {
       const {
         speech: {
-          options: { voiceURI, pitch, rate }
+          options: { voiceURI, pitch, rate },
+          elevenLabsApiKey
```

Comment:
I don't think you have to get the `elevenLabsApiKey` from props here as you are passing the values in the function arguments